lib/document/document-converter.js

```diff
@@ -486,10 +486,22 @@ ${faDom.css()}
     if (node.getDocumentTitle()) {
       const doc = node.getDocument()
       if (this.hasTitlePage(doc)) {
-        return `<div id="cover" class="title-page front-matter">
-  <h1>${node.getDocumentTitle()}</h1>
-  <h2>${node.getDocument().getAuthor()}</h2>
+        const docTitle = node.getDocument().getDocumentTitle({ partition: true })
+        //If the document has a subtitle, we split title in h1 / subtitle in h2
+        if (docTitle.hasSubtitle()){
+          return `<div id="cover" class="title-page front-matter">
+  <h1>${docTitle.getMain()}</h1>
+  <h2>${docTitle.getSubtitle()}</h2>
+  <div id="doc-author">${node.getDocument().getAuthor()}</div>
+  <div id="doc-rev-infos">Version ${node.getDocument().getRevisionNumber()}, ${node.getDocument().getRevisionDate()}</div>
```

Comment:
I think the HTML structure should be:

```js
<div class="details">
<span class="author">${node.getDocument().getAuthor()}</span>
<span class="revnumber"><!-- ... --></span>
<span class="revremark"><!-- ... --></span>
</div>
```

For the revision date, we could use `<time>` with `datetime` (if the date is valid).
Also keep in mind that an AsciiDoc document can have more than on author.
We should probably use `getAuthors` and display author's email if available.

For inspiration, you can take a look at:

- https://github.com/jirutka/asciidoctor-html5s/blob/5c53fd2749ad980b9015f0502a822f86f01a4f8b/data/templates/_header.html.slim#L5-L25
- https://github.com/asciidoctor/asciidoctor/blob/e0c8cc5c4398d37d2558063d25f6c8f100cf50b5/lib/asciidoctor/converter/html5.rb#L190-L213